src/client/src/types/trades.ts

```diff
@@ -0,0 +1,5 @@
+import { Trade } from './trade'
```

Comment:
the types package is a bit of an anti-pattern. We should not add new things to it. How many places is this interface used. If the answer is one and its put in nearer to the code that is using it.